### PR TITLE
meteor lake dgpu documentation

### PR DESCRIPTION
**Pull request by philipandag**

### CODE REVIEW

File: docs/unified/novacustom/overview.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ Select your Dasharo firmware flavor:
 
         </div>
 
-    === "V560TU"
-        __NovaCustom V560TU__ are 16 inch laptops based on 14th Generation Intel
+    === "V56 Series"
+        __NovaCustom V560Tx__ are 16 inch laptops based on 14th Generation Intel
```
Comment (mkopec) [inline docs/unified/novacustom/overview.md:23]:
```suggestion
        __NovaCustom V560TNx__ are 16 inch laptops based on 14th Generation Intel
```

File: docs/unified/novacustom/overview.md [hunk 2]
```diff
@@ -7,8 +7,8 @@ Select your Dasharo firmware flavor:
 
     ## Models
 
-    === "V540TU"
-        __NovaCustom V540TU__ are 14 inch laptops based on 14th Generation Intel
+    === "V54 Series"
+        __NovaCustom V540Tx__ are 14 inch laptops based on 14th Generation Intel
```
Comment (mkopec) [inline docs/unified/novacustom/overview.md:11]:
```suggestion
        __NovaCustom V540TNx__ are 14 inch laptops based on 14th Generation Intel
```

**Review by mkopec — COMMENTED** [review]
2 minor comments

File: docs/variants/novacustom_v560tnx/hardware-matrix.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+| **RAM**                        | Slot 1: W-NM56S516G                              |
+| **SSD**                        | Goodram PX700 1TB                                |
+| **Flash memory**               | GigaDevice 25LB256FYIG                           |
+| **USB pendrives**              | SanDisk Ultra USB 3.0 32 GB                      |
+| **USB Keyboard**               | Logitech, Inc. Keyboard K120                     |
+| **Wireless card 1**            | Intel Wi-Fi 6E AX211                             |
+| **Wireless card 2**            | Intel Wi-Fi 7 BE200                              |
+| **Display**                    | Display 1: HDMI 1920x1080p                       |
+| **Network**                    | Local network wired connection                   |
+| **Internal devices**           | 1. 2880x1800 14 inch screen                      |
```
Comment (mkopec) [inline docs/variants/novacustom_v560tnx/hardware-matrix.md:73]: Is that correct? V56 is the 16 inch model ;) Please check the internal panel resolution (e.g. with xrandr)

Comment (philipandag) [reply docs/variants/novacustom_v560tnx/hardware-matrix.md:73]: Sorry, copy pasta ;-;

Comment (philipandag) [reply docs/variants/novacustom_v560tnx/hardware-matrix.md]: [8a5dcc2](https://github.com/Dasharo/docs/pull/870/commits/8a5dcc222026e52e24aec6efd26824e5f82653ae)

**Review by mkopec — APPROVED** [review]

**Pull request merged**